freqtrade/constants.py

```diff
@@ -33,7 +33,8 @@
 AVAILABLE_PAIRLISTS = ['StaticPairList', 'VolumePairList',
                        'AgeFilter', 'OffsetFilter', 'PerformanceFilter',
                        'PrecisionFilter', 'PriceFilter', 'RangeStabilityFilter',
-                       'ShuffleFilter', 'SpreadFilter', 'VolatilityFilter']
+                       'ShuffleFilter', 'SpreadFilter', 'VolatilityFilter',
+                       'ExternalPairList']
```

Comment:
No this is no longer used. If you'd like to have a follower have the exact same pairlist as a Leader, then best run StaticPairlist. However, the DataProvider makes available what the Leader's pairlist is, by using `dp.get_producer_pairs(producer_name='default')` so you can use it inside of your strategy.

Reply:
"Inside the strategy" you mean in "bot_loop_start()"? If you mean inside `populate_indicators() then this means that the bot will `get_producer_pairs()` over and over again redundantly. 

This will be a good point to cover in the documentation.

Reply:
i'm ... not entirely happy with the removal of ExternalPairlist - However i think it's best left to a future PR - as it's touching things we should look at seperately.

The data is made available to the dataprovider - the problem currently is that pairlists don't have access to this.
Adding this is quite simple in theory - but should be done as a separate, followup PR.

in the end, this means this pr will not fully cover "everything" - but it doesn't have to - as it provides the framework and tools for things like the pairlist to build upon in the future.